solr/CHANGES.txt

```diff
@@ -80,6 +80,8 @@ Optimizations
 
 * SOLR-9359: Enable warming queries to be managed using Config API (Andy Webb, Eric Pugh)
 
+* SOLR-16296: Remove the use of XmlConfigFile by QueryElevationComponent. (Haythem Khiri)
```

Comment:
Can you move this out of the "Optimizations" section and into "Other"?  In the end this is some misc. refactoring that improves the code but is completely invisible to users.